src/oops/SOLID/openClosePrinciple/before/taxes/InternTaxPayable.java

```diff
@@ -0,0 +1,19 @@
+package oops.SOLID.openClosePrinciple.before.taxes;
+
+import oops.SOLID.openClosePrinciple.before.employees.Employee;
+
+public class InternTaxPayable implements TaxPayable{
+
+    private final double INCOME_TAX_PERCENT = 15;
+
+    @Override
+    public double calculateTax(Employee employee) {
+
+        int monthlySalary = employee.getMonthlyIncome();
+        int annualSalary = monthlySalary * 12;
```

Comment:
calculation for annualSalary is something related to employee and therefore, could move to Employee class.

Reply:
Added it here since this is something related to the business logic for Intern calculation and thought it would become repetitive if you have different flavours of getter for the same attribute.